proposals/csharp-10.0/GlobalUsingDirective.md

```diff
@@ -107,13 +102,11 @@ A C# program consists of one or more compilation units, each contained in a sepa
 
 The *global_using_directive*s of a compilation unit affect the *global_attributes* and *namespace_member_declaration*s of all compilation units in the program.
 
-## Extern aliases
-https://github.com/dotnet/csharplang/blob/master/spec/namespaces.md#extern-aliases
+## Extern aliases [§13.4](https://github.com/dotnet/csharpstandard/blob/draft-v6/standard/namespaces.md#134-extern-alias-directives)
```

Comment:
I'm surprised that we would use a folder named `draft-v6`. Does that mean we'll need to update all those links again when it's no longer a draft?
An alternative would be to place the draft in a folder called `v6` and put a "DRAFT" disclaimer at the top.

Reply:
`draft-v6` is the branch name. (It is the current default branch for that repo). The plan for the committee is to rename that branch `v6-standard` when the updated standard is submitted to ECMA for approval. By renaming the branch, all these links would resolve to same text.

By linking to that branch, we make sure these links continue to resolve to the version of the spec from which the proposals are based. In other words, once these proposals have been incorporated into the text, the feature spec links remain relevant historical references rather than pointing to the updates in the standard.